src/Clap-Core.package/ClapFlagGroup.class/properties.json

```diff
@@ -0,0 +1,11 @@
+{
+	"commentStamp" : "",
+	"super" : "ClapGroup",
+	"category" : "Clap-Core-Parameters",
```

Comment:
groups and categories are not parameters… make another category, e.g. `Clap-Core-Documentation`